src/inc/msquic.h

```diff
@@ -598,7 +600,8 @@ typedef struct QUIC_SETTINGS {
             uint64_t ServerResumptionLevel                  : 1;
             uint64_t MaxOperationsPerDrain                  : 1;
             uint64_t MtuDiscoveryMissingProbeCount          : 1;
-            uint64_t RESERVED                               : 33;
+            uint64_t IdleSrcCidChangeMs                     : 1;
```

Comment:
Following the pattern set by other times outs (such as `SendIdleTimeoutMs`) I think we should rename to `SrcCidUpdateIdleTimeoutMs`. Thoughts?

Reply:
I felt in the same way, renamed